src/docs/quick-start.md

```diff
@@ -2,196 +2,167 @@
 
-In this guide, we will deploy a simple To-Do App running [NextJS](https://nextjs.org/docs/getting-started) and [Prisma](https://www.prisma.io/docs/) using a template.
+**In this guide we will cover two different topics to get you quickly started with the platform -**
 
-<Image src="/images/getting-started/1-railway-getting-started.gif"
-alt="Preview of What The Guide is Building"
-layout="intrinsic"
-width={800} height={455} quality={100} />
+1. **Deploying your project** - Bring your code and let Railway handle the rest.
 
-We will get you up and running quickly by covering the following three steps:
+    **[Option 1](/quick-start#deploying-your-project---from-github)** - Deploying from GitHub.
 
-1. Creating a project from a template
-2. Developing the project locally
-3. Deploying to the cloud
+    **[Option 2](/quick-start#deploying-your-project---with-the-cli)** - Deploying with the [CLI](/guides/cli).
 
-You will need [Node](https://nodejs.org/en/download/) to run the project locally. And you will need either Node or [Homebrew](https://brew.sh/) on your machine to install the Railway CLI. We will use [Yarn](https://yarnpkg.com/) for our examples locally.
+3. **Deploying a template** - Ideal for deploying pre-configured software with minimal effort.
 
-## Create a Project
+For our project, we'll utilize a basic <a href="https://github.com/railwayapp-templates/nextjs-basic" target="_blank">NextJS app</a>, and for the template deployment, we'll use the <a href="https://railway.app/template/umami-analytics" target="_blank">Umami template</a>.
 
-First, to deploy the NextJS Prisma template, we will make a new [project](develop/projects).
+## Deploying Your Project - From GitHub
 
-Railway offers a Command Palette that exposes all actions that one can do on the platform. We will use this menu to create our project.
+To deploy our NextJS app, we will make a new <a href="/overview/the-basics#project--project-canvas" target="_blank">project</a>.
 
-Press the Command + K key combination and type "New Project".
+- Open up the <a href="/overview/the-basics#dashboard--projects" target="_blank">dashboard</a> → Click **New Project**.
 
-<Image src="/images/getting-started/2-railway-getting-started-create-a-project.gif"
-alt="Command K in Action"
-layout="intrinsic"
-width={800} height={455} quality={80} />
+- Choose the **GitHub repo** option.
 
-Under the list of options in the menu, select "Deploy Template".
-
-### Deploy the Next JS Prisma Template
+<Image src="https://res.cloudinary.com/railway/image/upload/v1723752559/docs/quick-start/new_project_uyqqpx.png"
+alt="screenshot of new project menu with deploy from github selected"
+layout="responsive"
+width={836} height={860} quality={100} />
 
-Railway offers 50+ templates ranging from blogs to self-hosted apps. In the Command Palette, type `NextJS Prisma`.
+*Railway requires a valid GitHub account to be linked. If your Railway account isn't associated with one, you will be prompted to link it.*
```

Comment:
lol keep reading Melissa.  still wonder about the fork step, might be good?  i'll go through the rest before i add comments 😄